tensornetwork/backends/base_backend.py

```diff
@@ -324,3 +324,15 @@ def eigsh_lanczos(
     """
     raise NotImplementedError(
         "Backend '{}' has not implemented eighs_lanczos.".format(self.name))
+
+  def multiply(self, tensor1: Tensor, tensor2: Tensor) -> Tensor:
+    """
+      Return the default multiplication of `tensor`.
+      A backend can override such implementation.
+      Args:
+        tensor1: A tensor.
+        tensor2: A tensor.
+      Returns:
+        Tensor
+    """
+    return tensor1 * tensor2
```

Comment:
I'd prefer to raise an NotImplementedError here instead and add it to all the separate backends even if the implementation is identical. Anyway, definitely we need a test for this wherever it ends up.

Reply:
+1